package.json

```diff
@@ -11,10 +11,11 @@
     "copy-env": "if ! [ -e .env ]; then cp .env.tpl .env; fi;"
   },
   "dependencies": {
+    "@material-ui/core": "^4.2.0",
```

Comment:
please use strict version like in other deps

Reply:
there is no need, yarn keep version strict via lock file